Fixed example, added screenshots and instructions for Studio (#1115)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/global-settings-configuration.adoc

Global settings for a mule application such as the default transaction timeout or
default error handler can be configured in the `<configuration>` element of the
root level of the xref:about-mule-configuration.adoc[Mule Configuration File]. +

> 

Reviewer: Edit instructions:

I have 3 PRs for you. I'll send them in separate messages so you can reply separately. These are no priority, so you can look at them either this or next week:
 https://github.com/mulesoft/docs-mule-runtime/pull/1115

== Create a Global Configuration from Studio

To create a global settings configuration:

. In Studio, select the *Global Elements* tab and click *Create*:
+
image::mruntime-global-settings-create.png[create-global-settings]
. Select *Global Configurations* > *Configuration*:
+
image:mruntime-global-configurations.png[configure-global-settings]
. Configure the *Default Error Handler* and the *HA Profile* values.

== Create a Global Configuration by Editing the XML

> The following example defines a global error handler, and then a global configuration which specifies a default transaction timeout and references the global error handler as the default one:

Reviewer: See "that" vs. "which" in style ref.:
```suggestion
The following example defines a global error handler, and then a global configuration that specifies a default transaction timeout and references the global error handler as the default:
```

[source,xml,linenums]
----
<mule>
  ...
  <!-- Error handler -->
  <error-handler name="GlobalErrorHandler"/>
  <!-- Global Settings -->
  <configuration doc:name="GlobalSettings" defaultTransactionTimeout="31337" defaultErrorHandler-ref="GlobalErrorHandler" />
  ...
  </configuration>
  ...
</mule>
----

> == Attributes of <configuration>

Reviewer: Is this topic and the one following supposed to be a child topic of the task above?

*Default*: none
|===

== Child Elements of <configuration>

[%header%autowidth.spread]
|===

> |Name |Description

Reviewer: Consider using list format instead of a table for two-column <thing>+<description> texts, especially when the second column runs longer than a sentence.
See "definition or description lists (dlists)" and "tables" in the style ref. for further guidance.
